### PR TITLE
Add a way to print an aliased without a trailing new line

### PR DESCRIPTION
**Pull request by jvoisin**
```
[0x00000000]> $v=1337
[0x00000000]> $v?; ?e lol
1337
lol
[0x00000000]> $v?n; ?e lol
1337lol
[0x00000000]>
```

### CODE REVIEW

**Comment by XVilka** [comment]
Long time no C!

**Comment by jvoisin** [comment]
Hence why it's a PR and not a commit on the master :D

File: libr/core/cmd.c [hunk 1]
```diff
@@ -542,7 +544,11 @@ static int cmd_alias(void *data, const char *input) {
 		*desc = 0;
 		char *v = r_cmd_alias_get (core->rcmd, buf, 0);
 		if (v) {
-			r_cons_println (v);
+			if (nonl == desc+1) {
```
Comment (radare) [inline libr/core/cmd.c]: Missing space around +

File: libr/core/cmd.c [hunk 2]
```diff
@@ -511,6 +512,7 @@ static int cmd_alias(void *data, const char *input) {
 	char *q = strchr (buf, ' ');
 	char *def = strchr (buf, '=');
 	char *desc = strchr (buf, '?');
```
Comment (radare) [inline libr/core/cmd.c]: The strchr seems like a very weak way to detect this, because this way the user cant echo a ? I think

Comment (jvoisin) [reply libr/core/cmd.c]: I wanted to do a minimal change.
I can refactor this if you prefer.

**Pull request merged**